caravel/assets/stylesheets/less/cosmo/bootswatch.less

```diff
@@ -252,18 +248,35 @@ label {
   &-default {
     .panel-heading {
       padding: 15px 15px 0 15px;
+      background-color: transparent;
     }
     .panel-title {
-      font-size: 14px;
-      font-weight: bold;
+      color: black;
+      padding-bottom: 5px;
+      border-bottom: 1px solid @gray-light;
+      h1, h2, h3, h4, h5, h6 {
+        margin: 10px 0 0 0;
+        font-weight: bold;
+      }
+    }
+    .close {
+      color: @text-color;
+    }
+  }
+  &-primary{
```

Comment:
it's handled by FAB... we'd have to fork or override it which I'd rather not do...